### ISSUE TITLE
add icons to session context menu

### ISSUE THREAD

**Issue by ronso0** [sienori/Tab-Session-Manager#576]
It happened a few times that I click the wrong button in the session context menu, for example "Open in current window" instead of "Add to current window"

In my experience icons help to avoid confusing menu items a lot, so let's add distinct icons to menu items.

**Comment by ronso0** [comment]
I can help with icon creation and implementation, I simply need a pointer where to start.

Is it as easy as assigning an Ids to each menu item in https://github.com/sienori/Tab-Session-Manager/blob/master/src/popup/components/SessionMenuItems.js#L51, then adding icons in https://github.com/sienori/Tab-Session-Manager/blob/master/src/popup/styles/Menu.scss ?

**Comment by sienori** [comment]
Menu icons are important for usability. But I didn't add icons because it was difficult to create...
If you could help me create an icon, I'd appreciate it!

To add an icon to the menu:
1. Add the icon (.svg) to the `popup/icons` folder
2. import and add the icon to `popup/components/SessionMenuItems.js`
```js
Import OpenInNewWindowIcon from "../icons/openInNewWindow.svg"

// ~~~

 return (
    <ul>
      // ~~~
      <li>
        <button onClick={handleOpenInNewWindow}>
          <OpenInNewWindowIcon /> {browser.i18n.getMessage("openInNewWindowLabel")}
        </button>
      </li>
```

Also, add a style of svg to `popup/styles/Menu.scss`
```.scss
#menu {
  // ~~~
  ul {
    // ~~~
    li {
      // ~~~
      button {
        // ~~~
        display: flex;
        align-items: center;

        svg {
          width: 16px;
          height: 16px;
          fill: var(--sub-text);
          margin-right: 5px;
        }

```

**Comment by ronso0** [comment]
Coming back to this after ages...
Me too had some trouble figuring out distinct icons for every action.

As stated above, my reason to add icons is to avoid messing up a saved session accidentally.
So maybe icons for the two sections `Open` and `Save / Edit` would be sufficient, maybe also a stronger separator?

**Comment by sienori** [comment]
In version 6.3.0, I added a save/edit icon to the menu.
![icon](https://user-images.githubusercontent.com/29514424/98445605-db4e1480-215b-11eb-9bb4-f8f48d390aff.png)

Also, Undo/Redo function is implemented. If you mess up an important session, you can undo it!

**Comment by ronso0** [comment]
Thanks! That improves the situation.

**Comment by RSTMP** [comment]
Hello sienori, I´m appreciating your work on this session manager, but can you please make it work, that the icons of the tabs are loaded after the restart of the session? And please install tab history. Maybe you can do categories as "last session", current session", "crashed session" and "saved session", as well?!
I´m no engineer or so, just wanted to let you know. I´m so angry, that no session manger works well for my use. I loved Michael Krafts session manager, it siuted me the best by far.
Just look at the structure it was build, before it was blocked by firefox: https://addons.thunderbird.net/De/seamonkey/addon/session-manager/
Please let me know what you think and if you will do some changes on your manager.
Greetings!

**Comment by ronso0** [comment]
@RSTMP These are entirely unrelated feature requests, just open a new Issue.

**Comment by RSTMP** [comment]
What is that good for? This thread is 2 years old. I just wanted to have an answer from @sienori.

**Comment by ronso0** [comment]
You have a feature request? File a feature requests, then it can be tracked and discussed. That's what Github Issues are for.
It helps staying foucsed aka 'on topic' ; )

**Comment by RSTMP** [comment]
Just did it...
